src/components/Chart/Report/GagueChart.vue

```diff
@@ -59,7 +59,7 @@
       >
         <b-form-textarea
           v-model="metric.fx"
-          placeholder="n"
+          :placeholder="$t('n')"
```

Comment:
The `n` here is a variable for the expression and shouldn't be translated. The same applies to the other `n` after this file.

Reply:
okay, reverting these changes